committeeoversightapp/views.py

```diff
@@ -82,7 +84,21 @@ class EventListJson(BaseDatatableView):
     max_display_length = 500
 
     def filter_queryset(self, qs):
-        # based on example at https://pypi.org/project/django-datatables-view/
+        # in order to filter by categories and committees, grab the detail
+        # object type and its pk from the url and filter
+        path = self.request.path
+        split_path = path.split('/')
```

Comment:
total aside: i have a weird bugaboo about avoiding accessing elements by their index, unless it's absolutely necessary. unpacking into multiple variables is my fave way to get around it, e.g., `a, b = 'some/url'.split('/')`. not a required change, esp if you make other changes to this route, just some fun trivia.